scep/scep.go

```diff
@@ -217,6 +224,11 @@ func ParsePKIMessage(data []byte, opts ...Option) (*PKIMessage, error) {
 		return nil, err
 	}
 
+	// According to RFC #2315 Section 9.1, it is valid that the server sends fewer
```

Comment:
Thanks for adding this clarification. Mind also describing what jscep does?

Reply:
Sure, done.